src/main/java/seedu/address/model/person/Person.java

```diff
@@ -22,19 +22,22 @@ public class Person {
     private final Email email;
 
     // Data fields
-    private final Address address;
     private final Set<Tag> tags = new HashSet<>();
 
+    private final int id;
+
+    public static int idTracker = 1;
+
     /**
      * Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
-        requireAllNonNull(name, phone, email, address, tags);
+    public Person(Name name, Phone phone, Email email, Set<Tag> tags, int id) {
+        requireAllNonNull(name, phone, email,  tags, id);
```

Comment:
What was the reason in which the id parameter is at the last position instead of the first? By convention, I believe ID is usually the first parameter

Reply:
Now the ID is "automatically" generated